creator-node/src/routes/files.js

```diff
@@ -168,7 +168,7 @@ module.exports = function (app) {
     try {
       // If client has provided filename, set filename in header to be auto-populated in download prompt.
       if (req.query.filename) {
-        res.setHeader('Content-Disposition', `inline; filename*=UTF-8''${req.query.filename}`)
+        res.setHeader('Content-Disposition', `attachment; filename*=UTF-8''${req.query.filename}`)
```

Comment:
why change this? reason it was `inline` was to match the response headers from the public gateway requests.